ksql-cli/src/test/java/io/confluent/ksql/cli/CliTest.java

```diff
@@ -859,6 +859,28 @@ public void shouldDescribeAggregateFunction() throws Exception {
     assertThat(output, containsString(expectedVariant));
   }
 
+  @Test
+  public void shouldDisplayInsertedKeyValue() throws Exception {
+
+    final String message = String.format(
```

Comment:
why use `format` here with constants? Just inline it!

Also, you may have noticed other tests using the format:

```java
// Given:

// When:

// Then:
```

This is a good structure for tests as it encourages clear separation any any setup (GIVEN), what's under test (WHEN) and when it expected (THEN).

```java
@Test
  public void shouldDisplayInsertedKeyValue() throws Exception {
      // Given:
      final String lines =
        "CREATE STREAM " + streamName + " (id INT, rating DOUBLE) WITH "
            + "(kafka_topic='ratings', partitions=1, value_format='json');\n"
        + "INSERT INTO " + streamName + "(id, rating) VALUES (294, 8.1);";

      // When:
     localCli.handleLine(lines);

      // Then:
     assertThat(terminal.getOutputString(), containsString("Inserted:BLAH));
  }
```

Note, I've removed the `dropStream` call  - ideally we'd put that in `tearDown`.

Reply:
There's a format as it was slightly easier to copy and paste that from the code, then write out a literal, but I'll change it to a literal.